requirements.txt

```
@@ -4,3 +4,4 @@ numpy>=1.19.5
 Pillow>=8.2.0
 python-magic>=0.4.22
 regex>=2021.4.4
+vidgear
```

FYI update: my `vidgear` PR https://github.com/abhiTronix/vidgear/pull/268 is now landed :) We just have to wait for the maintainer of that library to upload a new `vidgear` package to pypi, which they said they will within a week!

---

```suggestion
vidgear>=0.2.4
```
v0.2.4 [is here](https://github.com/abhiTronix/vidgear/pull/275) with the trimmed down dependencies!!